src/php/views/partials/content-single.twig

```diff
@@ -1,8 +1,19 @@
-<article id="post-{{ post.id }}">
-	<h2>
-		{{ post.title }}
-	</h2>
+<article id="post-{{ post.id }}" class="post-preview">
+	{% if post.thumbnail %}
+		<img
+			src="{{ post.thumbnail.src('thumbnail') }}"
+			class="post-preview__featured-image"
+			{% if post.thumbnail.alt and post.thumbnail.alt|length > 0 %}
+				alt="{{ post.thumbnail.alt }}"
+			{% endif %}
+		/>
+	{% endif %}
 	<div>
-		{{ post.preview }}
+		<h2 class="post-preview__title">
+			{{ post.title }}
+		</h2>
+		<div>
+			{{ post.preview.chars(180) }}
```

Comment:
Timber says that the default preview length is 50 words but in randomly spot-checking some of my posts I wasn't sure if this is accurate. I wanted to make the post previews more uniform in length so I went for setting a limit on the number of characters instead. Spot-checking again it still seems like there still are some (longer than 180 char) posts that preview less than 180 characters, but it's a little bit more consistent 🤷‍♀️

Reply:
I generated a list of numbers to test this, removing the `.chars(180)`, and it looks like it truncated right at 50 words:
![image](https://github.com/sparkbox/sparkpress-wordpress-starter/assets/565751/6543ffdc-0b0c-488f-bf85-e0f57bffe406)

There are probably edge cases, but I think we can stick with the default behavior for this PR, and leave it up to the user to determine how to truncate.